react/features/recent-list/functions.web.js

```diff
@@ -12,7 +12,7 @@ import { parseURIString, safeDecodeURIComponent } from '../base/util';
  */
 export function toDisplayableList(recentList) {
     return (
-        recentList.slice(-3).reverse()
+        recentList.slice(-1 * interfaceConfig.RECENT_LIST_DEPTH).reverse()
```

Comment:
This will not work for those that do not have interface config updated and will break their deployments.